jscomp/build_tests/super_errors/expected/warnings5.res.expected

```diff
@@ -0,0 +1,84 @@
+
+  [1;33mWarning number 9[0m
+  [36m/.../fixtures/warnings5.res[0m:[2m12:3-21[0m
+
+  10 [2m│[0m 
+  11 [2m│[0m switch y {
+  [1;33m12[0m [2m│[0m | [1;33m{otherValue: false}[0m => Js.log("first")
+  13 [2m│[0m }
+  14 [2m│[0m 
+
+  the following labels are not bound in this record pattern: someValue, typ
+Either bind these labels explicitly or add ', _' to the pattern.
+
+
+  [1;33mWarning number 8[0m
+  [36m/.../fixtures/warnings5.res[0m:[2m11:1-13:1[0m
+
+   9 [2m│[0m @val external y: someRecord = "otherVariable"
+  10 [2m│[0m 
+  [1;33m11[0m [2m│[0m [1;33mswitch y {[0m
+  [1;33m12[0m [2m│[0m [1;33m| {otherValue: false} => Js.log("first")[0m
+  [1;33m13[0m [2m│[0m [1;33m}[0m
+  14 [2m│[0m 
+  15 [2m│[0m switch y {
+
+  You forgot to handle a possible case here, for example: 
+  {otherValue: true}
+
+
+  [1;33mWarning number 9[0m
+  [36m/.../fixtures/warnings5.res[0m:[2m16:3-26[0m
+
+  14 [2m│[0m 
+  15 [2m│[0m switch y {
+  [1;33m16[0m [2m│[0m | [1;33m{typ: WithPayload(true)}[0m => Js.log("first")
+  17 [2m│[0m }
+  18 [2m│[0m 
+
+  the following labels are not bound in this record pattern: someValue, otherValue
+Either bind these labels explicitly or add ', _' to the pattern.
+
+
+  [1;33mWarning number 8[0m
+  [36m/.../fixtures/warnings5.res[0m:[2m15:1-17:1[0m
+
+  13 [2m│[0m }
+  14 [2m│[0m 
+  [1;33m15[0m [2m│[0m [1;33mswitch y {[0m
+  [1;33m16[0m [2m│[0m [1;33m| {typ: WithPayload(true)} => Js.log("first")[0m
+  [1;33m17[0m [2m│[0m [1;33m}[0m
+  18 [2m│[0m 
+  19 [2m│[0m let arr = [1]
+
+  You forgot to handle a possible case here, for example: 
+  ({typ: WithPayload(false)}|{typ: Variant})
```

Comment:
This looks a little dense.
Aslo, parens probl non needed.

Reply:
@cristianoc check the latest commit, what do you think?